src/components/ToggleRefinement.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div
+    v-if="state"
+    :class="[suit(''), !canRefine && suit('', 'noRefinement')]"
+  >
+    <slot
+      :value="state.value"
+      :can-refine="canRefine"
+      :refine="state.refine"
+      :create-URL="state.createURL"
+    >
+      <label :class="suit('label')">
+        <input
+          :class="suit('checkbox')"
+          type="checkbox"
+          :name="state.value.name"
+          :value="on"
+          :checked="state.value.isRefined"
+          @change="state.refine(state.value)"
+        />
+        <span :class="suit('labelText')">
+          {{ state.value.name }}
+        </span>
+        <span
+          v-if="state.value.count !== null"
+          :class="suit('count')"
+        >
+          {{ state.value.count.toLocaleString() }}
+        </span>
+      </label>
+    </slot>
+  </div>
+</template>
+
+<script>
+import { connectToggle } from 'instantsearch.js/es/connectors';
+import algoliaComponent from '../component';
+
+export default {
+  mixins: [algoliaComponent],
+  props: {
+    attribute: {
+      type: String,
+      required: true,
+    },
+    label: {
+      type: String,
+      required: true,
+    },
+    on: {
+      type: [String, Number, Boolean],
+      required: false,
```

Comment:
Did you make it so neither are required so that you can choose to only have off? 

I'd add a `validator` that checks that at least one of both is given, does that make sense?

Reply:
With the current implementation of the connector none is required. By default the `on` value is `true` and the `off` value is nothing. It means that you can provide none of them, `on`, `off` or both at the same time. That's why I put the `on` as non required. Is it more clear?

Reply:
ah yes, with a default value it makes sense